analyzer/codechecker_analyzer/analyzers/clangsa/analyzer.py

```diff
@@ -414,14 +411,18 @@ def construct_config_handler(cls, args, context):
                                'the command line for %s', cls.ANALYZER_NAME)
             cmdline_checkers = None
 
-        handler.initialize_checkers(
+        init_warn = handler.initialize_checkers(
             context.available_profiles,
             context.package_root,
             checkers,
-            clang_sa_checkers,
+            profile_configs,
             cmdline_checkers,
             'enable_all' in args and args.enable_all)
 
+        if init_warn:
+            LOG.warning("There was a problem during initializing the "
+                        "the checkers for clangsa!")
```

Comment:
Yes I know this does not tell much here, but if there is some warnings they are printed during the initialization.
The current initialization logic is not the best from the error handling point of view, because some initialization steps should be executed even it some previous step failed. It would take much more work to refactor.
If the default profile does not contain any checkers the initialization still can be successful if the user enabled some checkers, this init logic merges the checkers from the default profile configs, and the user who can set multiple profiles or checkers.